_docs/for-contributors/2017-01-01-building-from-source.md

```diff
@@ -39,7 +47,31 @@ Switch to the latest release branch, for example, `cardano-sl-1.0`:
 
     $ git checkout cardano-sl-1.0
 
-Then enter `nix-shell`:
+## Pure Nix build mode (recommended)
+
+Two simple steps:
+
+1.  To employ the signed IOHK build cache:
```

Comment:
it's called binary cache

Reply:
Depends on the perspective -- mixing usage of both terms might help newcomers to establish conceptual mapping.  But I'm agnostic.

Reply:
Done.